packages/blockchain-wallet-v4-frontend/src/scenes/app.tsx

```diff
@@ -143,6 +143,9 @@ const App = ({
 }
 
 const mapStateToProps = (state) => ({
+  apiUrl: selectors.core.walletOptions.getDomains(state).getOrElse({
+    api: 'https://api.dev.blockchain.info'
```

Comment:
probably want to prefer prod on fallback.

Reply:
hmm, good point!